packages/core/src/components/way-select/way-select.scss

```diff
@@ -0,0 +1,101 @@
+.w2w-select {
+  $height: 40px;
+  $color-selected: rgba(133, 172, 226, 1);
```

Comment:
But his is something that could me improved later in a separate PR

Reply:
sass functions like darken() cannot read custom properties because they are assigned at browser runtime.

I'm fine with using custom properties but we will lose the sass functionalities over it.

Reply:
Yeah that's the only downside. But Custom Properties are the only way to customise the Web Components (and Shadow Parts).